tools/container/toolkit/toolkit.go

```diff
@@ -342,7 +357,14 @@ func Install(cli *cli.Context, opts *Options, toolkitRoot string) error {
 		log.Errorf("Ignoring error: %v", fmt.Errorf("error installing NVIDIA Container CDI Hook CLI: %v", err))
 	}
 
-	err = installToolkitConfig(cli, toolkitConfigPath, nvidiaContainerCliExecutable, nvidiaCTKPath, nvidiaContainerRuntimeHookPath, opts)
+	var runtimeBinPaths []string
+	runtimeBinPaths, err = getRuntimeBinaryPaths(hostRoot)
+	if err != nil {
+		log.Warningf("Error retrieving runtime binary paths: %v", err)
+	}
```

Comment:
I would prefer if we pull this up to outside of the toolkit and pass this in as an argument to `Install` as a field in `Options` or as a dedicated parameter. I don't want this config file interaction hidden.

Reply:
`runtimeBinPaths` is calculated from the input parameters to the Install method. I wouldn't consider this as a `hidden behaviour`. It is, in fact, a part of the business logic performed with the parameters/options passed to the Install method